lib/core/include/irods/irods_service_account.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef __IRODS_SERVICE_ACCOUNT_HPP__
+#define __IRODS_SERVICE_ACCOUNT_HPP__
+
+namespace irods
+{
+    /// Attempts to determine whether the caller is running as the irods service unix account.
+    ///
+    /// \since 4.3.1
```

Comment:
A long-standing question I've always had...

Do we put 4.2.12 here, or is 4.3.1 right for main/4-3-stable and 4.2.12 is just for 4-2-stable?

Reply:
since it wasn't in 4.3.0... putting 4.2.12 is a tad misleading.

Reply:
There's no reason we can't put both. From doxygen docs:
> The paragraph that follows \since does not have any special internal structure.

Reply:
I normally put the earliest release where it is available.

Reply:
Putting both works well considering it isn't in 4.3.0.